src/core/lib/security/credentials/s2a/grpc_s2a_credentials_options.cc

```diff
@@ -79,10 +57,10 @@ grpc_s2a_credentials_options* grpc_s2a_credentials_options::copy() const {
 }
 
 bool grpc_s2a_credentials_options::check_fields(
```

Comment:
Please change the function name to follow the C++ naming style - "Regular functions have mixed case; accessors and mutators may be named like variables. Ordinarily, functions should start with a capital letter and have a capital letter for each new word."

Reply:
This applies to all other functions defined in this PR.

Reply:
Done in next commit.